lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/Context/ContextRuntime.cs

```diff
@@ -331,7 +332,7 @@ public void DeliverTaskMessage(byte[] message)
             }
         }
 
-        public void HandleContextMessaage(byte[] mesage)
+        public void HandleContextMessage(byte[] mesage)
```

Comment:
Oh, right. I completely forget about that since that was about #697 instead of this PR.
Anyway, both PRs are about `ContextRuntime.cs`. Thank you for reminding that. :)